src/guardian/guardian.service.ts

```diff
@@ -231,64 +286,21 @@ export class GuardianService implements OnModuleInit {
    * @param blockData - collected data from the current block
    * @returns list of keys that were deposited earlier
    */
-  public getNextKeysIntersections(
-    blockData: BlockData,
-  ): VerifiedDepositEvent[] {
+  public getKeysIntersections(blockData: BlockData): VerifiedDepositEvent[] {
```

Comment:
I think we should explicitly put in the name that the intersections are searched for one module. As in all other methods that have the same behavior

Reply:
There is also a problem with the semantic of the argument. It's called `blockData`, but it contains data for one module + general data. Consider splitting the data into several arguments